### PR TITLE
Rename ocp-qe-perfscale-ci-tests to control-plane

### PR DESCRIPTION
**Pull request by krishvoor**
This PR renames ocp-qe-perfscale-ci-tests --> Control-Plane
& updates the required jobs accordingly.

### CODE REVIEW

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ tests:
     test:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-azure-aro
-- as: ocp-qe-perfscale-ci-tests-6nodes-aro
+- as: ocp-qe-perfscale-cp-6nodes-aro
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:55]: @krishvoor Can we add periodic at the end of the name here?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml [hunk 2]
```diff
@@ -35,9 +35,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: openshift-qe-ocp-qe-perfscale-ci-tests
+    workflow: openshift-qe-control-plane
   timeout: 5h0m0s
-- as: ocp-qe-perfscale-aws-ci-tests-small-scale-kube-burner
+- as: ocp-qe-perfscale-aws-cp-small-scale-kube-burner
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:40]: Same here, can we rename "small scale" to actual number of nodes like other tests?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml (hunk 1 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:55]: Sure, will do

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-aws-ci-tests-medium-scale-kube-burner
+- as: ocp-qe-perfscale-aws-cp-medium-scale-kube-burner
```
[the remark this answers is not in the document]
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:24]: Good idea, will update

**Comment by svetsa-rh** [comment]
@krishvoor CNO team will start noticing their tests failing after this change. Can we give them a heads up? TIA!

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-ec-perfscale-rosa-multiaz-ec-cp-2nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-cp-6nodes-aro-periodic

**Comment by krishvoor** [comment]
Blocks:
https://github.com/openshift/release/pull/42636 
https://github.com/openshift/release/pull/42639

**Comment by krishvoor** [comment]
@dcbw can you ptal, this just updates step-registry `ocp-qe-perfscale-ci-tests` --> `control-plane`

**Comment by svetsa-rh** [comment]
@krishvoor Harsha, did you rebase with latest code? I think @jtaleric recently changed all these references in CNO and only calls specific workload such as `cluster-density-v2`

**Comment by krishvoor** [comment]
> @krishvoor Harsha, did you rebase with latest code? I think @jtaleric recently changed all these references in CNO and only calls specific workload such as `cluster-density-v2`

Just did a rebase a while ago, things are good to go.

**Comment by dcbw** [comment]
The changes look OK, but aren't these fundamentally still perf/scale tests? I guess it's a bit confusing to have them called "qe-control-plane" which doesn't indicate that they are perf/scale tests, and not functional/integration/etc tests that QE also does.

I'm fine with the change if this is really what you intend, just wondering if there might be confusion outside of the QE/PS teams in the future with this name.

**Comment by krishvoor** [comment]
> The changes look OK, but aren't these fundamentally still perf/scale tests? I guess it's a bit confusing to have them called "qe-control-plane" which doesn't indicate that they are perf/scale tests, and not functional/integration/etc tests that QE also does.

QE & PerfScale are jointly collaborating and capturing the ongoing work here [step-registry](https://github.com/openshift/release/tree/master/ci-operator/step-registry/openshift-qe/) & [job-configs](https://github.com/openshift/release/tree/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci)
Prow has a strict checking policy, to comply with it we are updating the job names respectively.


> I'm fine with the change if this is really what you intend, just wondering if there might be confusion outside of the QE/PS teams in the future with this name.

Of course, this entire effort of updating the job/step-registry's naming nomenclature is to bring standardization.
In the future, those job names will bring more clarity.

@dcbw thanks for taking the time to review/comment.
Can you also please `/lgtm` so we can progress accordingly?

**Comment by svetsa-rh** [comment]
@dcbw Dan, the same thought occurred to me yesterday whether others (outside perfscale)will get confused just looking at the names. I will be opening a new PR soon to rename the tests. Hopefully that will clarify more.
The tests will be in the same location though.

**Pull request merged**